### PR TITLE
Merge-RELEASE. Project 2. #1. Week 1

### PR DESCRIPTION
**Pull request by saffchen**

### CODE REVIEW

File: stock-system/src/main/resources/participants.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[
```
Comment (pelmegov) [inline stock-system/src/main/resources/participants.json:1]: We don't need participants.txt anymore if we are going to use this json. I would suggest to remove txt file.

File: vue-app/src/HomePage.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      name: 'app',
+      components: {
+          Header,
+          Footer,
+          Main
+      },
+      mounted() {
+        window.addEventListener("load", () => {
+          $('#datatable').DataTable();
+          document.getElementById('date').innerHTML = date();
```
Comment (pelmegov) [inline vue-app/src/HomePage.vue:41]: "date" can be like a variable in the component which has to show that date. It's not current component's responsibility.

File: vue-app/src/HomePage.vue [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  export default {
+      name: 'app',
+      components: {
+          Header,
+          Footer,
+          Main
+      },
+      mounted() {
+        window.addEventListener("load", () => {
+          $('#datatable').DataTable();
```
Comment (pelmegov) [inline vue-app/src/HomePage.vue:40]: All product table preparations should be in the specific component, it is not great idea to do it in parent component.

File: stock-system/src/main/resources/application.properties [hunk 4]
```diff
@@ -0,0 +1,8 @@
+server.port=8080
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```
Comment (fstk1337) [inline stock-system/src/main/resources/application.properties:3]: I think datasource target database in url should be 'stock', not 'postgres'.

File: stock-system/src/main/resources/participants.json (hunk 1 above)
Comment (saffchen) [reply stock-system/src/main/resources/participants.json:1]: Fixed

File: stock-system/src/main/resources/participants.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  {
+    "email": "denis.savchenko98@yandex.ru",
+    "nickName": "saffchen"
```
Comment (fstk1337) [inline stock-system/src/main/resources/participants.json:4]: In our vue-app we use 'email' and 'nickname' (second 'n' in lowercase) object keys. I suggest editing for further components compatibility.

**Review by fstk1337 — APPROVED** [review]
Good job)

File: stock-system/src/main/resources/participants.json (hunk 5 above)
Comment (saffchen) [reply stock-system/src/main/resources/participants.json]: Fixed

File: vue-app/src/HomePage.vue (hunk 2 above)
Comment (fstk1337) [reply vue-app/src/HomePage.vue:41]: fixed.

File: vue-app/src/HomePage.vue (hunk 3 above)
Comment (fstk1337) [reply vue-app/src/HomePage.vue:40]: fixed.

File: stock-system/src/main/resources/application.properties (hunk 4 above)
Comment (saffchen) [reply stock-system/src/main/resources/application.properties]: Fixed

**Pull request merged**